src/breadcrumbs/types.js

```diff
@@ -13,12 +13,14 @@ import type {OverrideT} from '../helpers/overrides.js';
 
 export type OverridesT = {
   Root?: OverrideT<*>,
-  Separator?: OverrideT<*>,
-  Icon?: OverrideT<*>,
```

Comment:
We can't remove the current functionality, this would likely break many applications. Can this change be implemented in a way that supports what currently exists?

Reply:
I did have an example of how to use this and implement the "best practice" markup by overriding the `Separator` to `null` but @gergelyke wondered if the pseudo behavior could be implemented as the default.